extension-manifest-v3/src/background/cosmetics.js

```diff
@@ -54,50 +56,45 @@ async function adblockerInjectStylesWebExtension(
     return;
   }
 
-  // Proceed with stylesheet injection.
-  return new Promise((resolve) => {
-    if (chrome.scripting && chrome.scripting.insertCSS) {
-      const target = {
-        tabId,
-      };
+  if (chrome.scripting && chrome.scripting.insertCSS) {
+    const target = {
+      tabId,
+    };
 
-      if (frameId) {
-        target.frameIds = [frameId];
-      } else {
-        target.allFrames = allFrames;
-      }
-      chrome.scripting.insertCSS(
-        {
-          css: styles,
-          origin: 'USER',
-          target,
-        },
-        () => resolve,
-      );
+    if (frameId) {
+      target.frameIds = [frameId];
     } else {
-      const details = {
-        allFrames,
-        code: styles,
-        cssOrigin: 'user',
-        matchAboutBlank: true,
-        runAt: 'document_start',
-      };
-      if (frameId) {
-        details.frameId = frameId;
-      }
-      chrome.tabs.insertCSS(tabId, details, () => resolve);
+      target.allFrames = allFrames;
     }
-  });
+    chrome.scripting.insertCSS({
```

Comment:
this returns a promise (catch and log should be good, we don't need to await it)